src/main/webapp/mentors.js

```diff
@@ -108,11 +108,11 @@ function createMentorElement(mentor) {
  * @param {String} email
  * @return {HTMLElement} mentorEmail
  */
-function createEmailLink(email) {
+export function createEmailLink(email) {
```

Comment:
Let's not export the functions if we don't need to. Here and below.
```suggestion
function createEmailLink(email) {
```